src/routes/(app)/events/calendar/Calendar.svelte

```diff
@@ -68,9 +68,11 @@
             start: event.startDatetime,
             end: event.endDatetime,
             url: eventLink(event),
+            color: event.isCancelled ? "rgb(250, 43, 43)" : "#f280a1",
+            className: event.isCancelled ? "!line-through" : "",
           })),
           locale: languageTag(),
-          eventColor: "#f280a1",
```

Comment:
It was moved into event specific properties (along with adding a check for isCancelled) so it will change to red if cancelled.
This line could be kept, with the check overriding the default. Which code style do we prefer?